src/libraries/Microsoft.Extensions.FileSystemGlobbing/src/Microsoft.Extensions.FileSystemGlobbing.csproj

```diff
@@ -2,18 +2,16 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <Nullable>enable</Nullable>
```

Comment:
why the lines below are changed?

Reply:
In the different nullable PR I'd added `$(NetCoreAppCurrent)` target framework, and was told that the way to reference dependencies is to add `DisableImplicitAssemblyReferences` instead of adding `<Reference>`, so applied it here as well.

https://github.com/dotnet/runtime/pull/57445#discussion_r695721810